### PR TITLE
profile tweaks

### PR DESCRIPTION
**Pull request by tsiology**
app_hr profile url arg changed from True|False to 1|0.

issued_total and user_balance displays rounded to 2 decimal places.

### CODE REVIEW

File: openCurrents/urls.py [hunk 1]
```diff
@@ -62,7 +62,7 @@
     url(r'^volunteering$', views.VolunteeringView.as_view(), name='volunteering'),
     url(r'^volunteer-requests$', views.VolunteerRequestsView.as_view(), name='volunteer-requests'),
     url(r'^profile$', views.ProfileView.as_view(), name='profile'),
-    url(r'^profile/(?P<app_hr>(True|False))/$', views.ProfileView.as_view(), name='profile'),
+    url(r'^profile/(?P<app_hr>(1|0))/$', views.ProfileView.as_view(), name='profile'),
```
Comment (nickolashe) [inline openCurrents/urls.py]: @tsiology can you remind me what we are using this param for?

Comment (dannypernik) [inline openCurrents/urls.py]: @nickolashe True if hours require approval & admin's first login of the week. True triggers approve-hours popup. @tsiology please confirm I've got that right

Comment (tsiology) [inline openCurrents/urls.py]: Yes, that is the functionality I observed, though I believe @karbmk first created this. I swapped the setting and the checking of True|False for 1|0.

Comment (nickolashe) [inline openCurrents/urls.py]: got it, thanks

**Pull request merged**